adbc.h

```diff
@@ -646,16 +646,16 @@ struct ADBC_EXPORT AdbcDriver {
   AdbcStatusCode (*ConnectionRelease)(struct AdbcConnection*, struct AdbcError*);
   AdbcStatusCode (*ConnectionRollback)(struct AdbcConnection*, struct AdbcError*);
 
-  AdbcStatusCode (*StatementBind)(struct AdbcStatement*, struct ArrowArray*,
-                                  struct ArrowSchema*, struct AdbcError*);
+  AdbcStatusCode (*StatementBind)(struct AdbcStatement*, struct ArrowArray const*,
+                                  struct ArrowSchema const*, struct AdbcError*);
```

Comment:
Binding should probably clear these structs since the statement is expected to take ownership